### PR TITLE
Added overlays' displacement debug visualization

### PR DESCRIPTION
**Pull request by rokuz**

### CODE REVIEW

**Comment by rokuz** [comment]
DUDE JTTP

File: drape/debug_rect_renderer.cpp [hunk 1]
```diff
@@ -106,12 +106,65 @@ void DebugRectRenderer::DrawRect(ScreenBase const & screen, m2::RectF const & re
...
+  m_program->Bind();
+
+  GLFunctions::glBindBuffer(m_vertexBuffer, gl_const::GLArrayBuffer);
+  GLFunctions::glBindVertexArray(m_VAO);
+
+  GLFunctions::glDisable(gl_const::GLDepthTest);
+
+  array<m2::PointF, 5> vertices;
+  m2::PointF dir = (data.m_arrowEnd - data.m_arrowStart).Normalize();
+  m2::PointF side = m2::PointF(-dir.y, dir.x);
```
Comment (darina) [inline drape/debug_rect_renderer.cpp]: const

**Comment by rokuz** [comment]
PTAL @darina

**Comment by rokuz** [comment]
DUDE JTTP

**Comment by darina** [comment]
LGTM

**Comment by vng** [comment]
LGTM

**Pull request merged**